src/ui/index.css

```diff
@@ -1,493 +1,135 @@
-    position: fixed;
-    left: 245px;
-    top: 61px;
-    padding: 10px; 
-    border-width: 4px;
-    border-radius: 8px;
+    width: calc(100% - 240px) !important;
+    height: calc(100% - 75px) !important;
+    position: fixed !important;
+    left: 240px !important;
+    top: 65px !important;
     overflow-y: auto;
+    overflow-x: hidden;
 }
 
-.json-input {
-    padding: 20px;
-    height: calc(100% - 50px);
-    font-family: monospace;
-    font-size: 1.1rem;
-    border: 2px solid rgb(212, 208, 208);
-    border-radius: 8px;
-    resize: none;
-    min-height: 150px;
-    overflow-y: auto;
-}
-
-.json-input:focus {
-    outline: none;
+.text-input {
+    min-height: 150px !important;
+    overflow-y: auto !important;
+    resize: none !important;
 }
 
-.enroll-button {
-    padding: 20px 20px;
-    margin-top: 10px;
-    font-size: 1.2rem;
-    background-color: #38103f;
-    color: white;
-    border: none;
-    cursor: pointer;
-    border-radius: 8px;
-    font-weight: bold;
+.text-input:focus {
+    outline: none !important;
+    box-shadow: none !important;
 }
 
-.enroll-button:hover {
-    background-color: #281548;
+.long-input {
+    height: calc(100vh - 165px) !important;
 }
 
-.dropdown-button {
-    padding: 10px 10px 10px 20px;
-    background-color: white;
-    margin-bottom: 10px;
-    width: 100%;
-    font-family: monospace;
-    font-size: 1.1rem;
-    border: 2px solid rgb(212, 208, 208);
-    border-radius: 8px;
-    resize: none;
-    color: #000000;
+.medium-input {
+    height: calc(100vh - 215px) !important;
 }
 
-.options {
-    align-items: center;
-    width: 100%;
-    display: flex;
+.submit-button {
+    background-color: #44144c !important;
 }
 
-.options__dropdown {
-    flex-basis: 85%;
-}
-
-.options__checkbox {
-    width: 20px;
-    height: 20px;
-    margin-right: 10px;
-    align-items: center;
-    display: flex;
-}
-
-.options__checkbox-label {
-    font-size: 1.1rem;
-    display: flex;
-}
-
-.options__checkbox-background {
-    padding: 10px 10px 10px 20px;
-    flex-basis: 25%;
-    display: flex;
-    align-items: center;
-    border: 2px solid rgb(212, 208, 208);
-    border-radius: 8px;
-    margin-left: 10px;
-    margin-bottom: 10px;
-}
-
-.settings {
-    padding: 10px;
-    height: 100%;
-    font-family: monospace;
-    font-size: 1.1rem;
-    resize: none;
-    margin-bottom: 3px;
-    overflow-y: auto;
-    overflow-x: auto;
+.submit-button:hover {
+    background-color: #281548 !important;
 }
 
-.settings__title {
-    color: #6a3d72;
-    font-size: 1.5rem;
-    font-weight: bold;
-    margin-top: 0px;
-    padding-left: 15px;
-    padding-top: 15px;
-    margin-bottom: 10px;
+.purple-underline {
+    border-top: 2px solid #6a3d72;
 }
 
-.settings__underline {
-    width: calc(100% - 30px);
-    border: 0;
-    border-top: 1px solid #6a3d72;
-    margin-left: 15px;
-    margin-bottom: 25px;
+.section-line {
+    border-top: 2px solid rgb(212, 208, 208);
```

Comment:
css custom props for all the colours please. And can we use hex for everything that doesnt require alpha channel?